torch/onnx/symbolic_opset9.py

```diff
@@ -2240,9 +2239,46 @@ def _std(g, input, dim, unbiased, keepdim):
 # torch.std(input, dim, keepdim=False, unbiased=True)
 def std(g, input, *args):
     if len(args) == 3:
-        return _std(g, input, *args)
+        var, _ = _var(g, input, *args)
```

Comment:
could this be simplified as return out, _ = var(g, input, *args); return g.op("Sqrt", out)

Reply:
Yeah I think this is more about style than code optimization.
Both styles have the same complexity.